### DISCUSSION TITLE
How to delete a branch.

### DISCUSSION THREAD

**Discussion by KrisKasprzak** [community/community#40740] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Somehow I have a branch (branching to someone else repository) in my repositories. Maybe I created a branch--can't remember.

How do I delete it? I'm able to delete all files and directories in the branch but no option to delete the repository.

Help?

**Comment by DSDmark** [comment]
Hi @KrisKasprzak,

- first Go to your **repository** and click on that branch as you can see number of branch showing on repository.

![image](https://user-images.githubusercontent.com/85744816/205479201-886e103c-14a0-46b8-b9d9-238f3d15aa83.png)

- After that, the page look like that. Here you can delete your branches or create new branches.

![image](https://user-images.githubusercontent.com/85744816/205479211-d7ff9abe-09a3-4fac-95ff-5c745c88b47c.png)

- if you want to delete your repository, then again open your repository and click on setting and scroll down.

![image](https://user-images.githubusercontent.com/85744816/205479230-61905304-6f45-433b-bca9-fb7e9015737d.png)

- Here you can see that section where you can delete your **repo**

![image](https://user-images.githubusercontent.com/85744816/205479240-3394d18f-ac41-4dd7-86d4-35b78af15646.png)

**Comment by mohamedzhioua** [comment]
hello @KrisKasprzak  here in the link below you can find the officiel  github documentation where you can find out how to delete the branch  :  "https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository"
or you can Open a [Git BASH window] or Command Window in the root of your Git repository
If necessary, use the git switch or checkout command to move off the branch you wish to delete

command to delete the local branch :
git branch --delete <branchname>
 
Run the git branch -a command to verify the local Git branch is deleted
